### PR TITLE
Small fixes here and there

### PR DESCRIPTION
**Pull request by KorigamiK**
Commit messages explain the gist.

### CODE REVIEW

File: chapters/chapter_2.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ This should land you in a text editor, usually vi(m), where you add commit title
 #
 ```
 
-This is giving the output you saw with `git status` one more time, so that you can be sure that you are committing all the files you want. On the first line, give a commit title. Leave a blank line, and from the third line, you can write the description.
+The is giving the output you saw with `git status` one more time, so that you can be sure that you are committing all the files you want. On the first line, give a commit title. Leave a blank line, and from the third line, you can write the description.
```
Comment (flyingcakes85) [inline chapters/chapter_2.md]: Probably not needed

**Review by flyingcakes85 — CHANGES_REQUESTED** [review]

**Comment by flyingcakes85** [comment]
Thanks for the proof read. Much needed :)

One question - is the file `vscode/ltex.hiddenFalsePositives.en-US.txt` required?

**Comment by KorigamiK** [comment]
It's not critical for development but you will have to ignore these warning once again. 
If you wish to purge these changes you could add them to your local settings.
But that's assuming you even use vs code 😅. So no, the entire `.vscode` directory does not affect the project.

File: chapters/chapter_2.md (hunk 1 above)
Comment (KorigamiK) [reply chapters/chapter_2.md]: It seems like the error was many consecutive sentences starting with "This". Should've corrected

**Review by flyingcakes85 — APPROVED** [review]

**Pull request merged**